src/test/resources/features/highlighting/inspections/multiline_string_args_inspetion.feature

```diff
@@ -4,19 +4,38 @@ Feature: Inspection for multiline arguments in same section
     Given I open a file "foo.smk" with text
     """
     <rule_like> NAME:
-        <section>: "a" "b", "c"
+        <section>: <args>, "c"
     """
     And SmkSectionMultilineStringArgsInspection inspection is enabled
-    Then I expect inspection warning on <"a" "b"> with message
+    Then I expect inspection weak warning on <<args>> with message
     """
     Multiline string argument in '<section>' will be considered as concatenation.
     """
-    When I check highlighting warnings
+    When I check highlighting weak warnings
     Examples:
-      | section    | rule_like  |
-      | input      | rule       |
-      | output     | rule       |
-      | log        | rule       |
-      | input      | checkpoint |
-      | output     | checkpoint |
-      | log        | checkpoint |
\ No newline at end of file
+      | section | rule_like  | args                                |
+      | output  | rule       | "a" \n        "b"                   |
```

Comment:
Cool, I didn't know that `\n` here will be treated as line break, not just string with `\` and `n` chars! I've checked on runtime, that indeed file text has linebreaks.